api/HTMLMediaElement.json

```diff
@@ -1706,10 +1709,19 @@
               "version_added": false
             },
             "opera": {
-              "version_added": true
+              "version_added": "29"
```

Comment:
Confirmed with https://mdn-bcd-collector.appspot.com/tests/api/HTMLMediaElement/mediaKeys in Opera 29. This was an API I expected shipped a few versions later in Opera, but no.